### ISSUE TITLE
Need of working login/signup page

### ISSUE THREAD

**Issue by singhrishabh93** [singhrishabh93/Hacktoberfest-2022#4]
Hey there!
There is a need of working login/signup project which should be added.
You can use any frontend/backend technology to build it.

Login can be done using email and password
Signup can be done using Name, email, password, and confirm password.

It must be responsive so that it can be viewed or accessed easily on any device.

Remember create pull requests only after 1 October.

**Comment by ganajayant** [comment]
hey,can i have this issue

**Comment by singhrishabh93** [comment]
> hey,can i have this issue

yes you can work on this issue but remember to pull this request only after 1st october

**Comment by utkarsh4321** [comment]
can i make it in express.js.

**Comment by singhrishabh93** [comment]
> can i make it in express.js.

yes, ofcourse

**Comment by ganajayant** [comment]
do you also want mongodb?

**Comment by singhrishabh93** [comment]
> do you also want mongodb?

If it is required you can use it.

**Comment by utkarsh4321** [comment]
ok

**Comment by ganajayant** [comment]
you just want login and signup with validation and backend rendering page? do want any authentication

**Comment by singhrishabh93** [comment]
> you just want login and signup with validation and backend rendering page? do want any authentication

No I don't want any authentication
I just want normal email and password validation

You can use authentication if you want to give your best.

**Comment by ganajayant** [comment]
so i will be using just static email and password for login is it okay?

**Comment by singhrishabh93** [comment]
> so i will be using just static email and password for login is it okay?

Yepp

**Comment by NandanVyas** [comment]
Hello @singhrishabh93, I have done something similar in my project and I would like to contribute to this issue if you assign me this.

**Comment by singhrishabh93** [comment]
> 

Yes ofcourse you can contribute to this issue

**Comment by singhrishabh93** [comment]
@utkarsh4321 @NandanVyas @ganajayant 
What's the progress?
Tell me separately.

**Comment by ganajayant** [comment]
i have completed the issue
i am waiting to make pr midnight after 12

**Comment by singhrishabh93** [comment]
> i have completed the issue
> i am waiting to make pr midnight after 12

Great man

**Comment by 6289513802** [comment]
I want to contribute in this issue can I? @singhrishabh93

**Comment by singhrishabh93** [comment]
> I want to contribute in this issue can I? @singhrishabh93 

Yes you can contribute

**Comment by singhrishabh93** [comment]
> i have completed the issue
> i am waiting to make pr midnight after 12

@ganajayant 
Can you share your hacktoberfest screenshot of my merged PR

**Comment by ganajayant** [comment]
it is showing your repo is excluded from hacktoberfest event
![image](https://user-images.githubusercontent.com/97283224/193403237-4ab7b151-3e10-48b2-b2b0-08587a009e89.png)
![image](https://user-images.githubusercontent.com/97283224/193403282-3bb2de51-c613-4660-8e7a-77d6edc46b82.png)

**Comment by singhrishabh93** [comment]
> it is showing your repo is excluded from hacktoberfest event ![image](https://user-images.githubusercontent.com/97283224/193403237-4ab7b151-3e10-48b2-b2b0-08587a009e89.png) ![image](https://user-images.githubusercontent.com/97283224/193403282-3bb2de51-c613-4660-8e7a-77d6edc46b82.png)

@ganajayant Still finding what's the problem with this repo...

**Comment by ganajayant** [comment]
once check guidelines of hacktoberfest that would help

**Comment by singhrishabh93** [comment]
> once check guidelines of hacktoberfest that would help

Can you do it for me??

**Comment by ganajayant** [comment]
yeah sure i will try

**Comment by singhrishabh93** [comment]
> yeah sure i will try

Have you figured it out what's the issue on my REPO

**Comment by ganajayant** [comment]
once look into this
https://hacktoberfest.com/participation/#pr-mr-details
